symbols.xml

```diff
@@ -219,22 +219,22 @@
         generated standingorders
 
         <global-address name='standing_orders_mix_food' value='0x00f9303d'/>
-        <global-address name='standing_orders_gather_bodies' value='0x00f9303e'/>
```

Comment:
I think 5 -6 of the ones in the original file were all the same address which is clearly wrong.  I'm not familiar with your code finding routines.

The standing_orders were found by locating the same 2-3 subroutines in both the 0.43.03 and 0.43.05 and comparing the order that the globals were referenced.  I then compared count of references to each address in other routines to verify.  Spot checked them by comparing routines from both versions where the code was used.